Stadgar.md

```diff
@@ -37,6 +39,8 @@ En gång per år hålls ett ordinarie årsmöte, normalt under mars månad. Extr
 
 Ett årsmöte kallas genom att två kallelser skickas till medlemmarna. Den första kallelsen ska ange tid och plats för mötet och får skickas tidigast två månader innan mötet. Den andra kallelsen ska skickas senast en vecka innan mötet. Den andra kallelsen ska innehålla dagordningen för mötet samt de handlingar som behövs för att fatta beslut under årsmötet.
 
+[/]:# (Tidpunkt för kallelse borde kanske ändras, beroende lite på vad syftet med tidsgränserna är. Om så många som möjligt ska kunna närvara, är det viktigt att kalla i god tid. Exempelvis skulle regeln kunna vara att den första kallelsen skickas tidigast två månader och senast en månad innan mötet.)
```

Comment:
Jag har faktiskt funderat på något liknande: Första kallelsen två till en månad innan. Denna tycker jag att du kan skriva in.